fenix/app/src/main/res/values/static_strings.xml

```diff
@@ -115,4 +115,7 @@
     <!-- Browser menu button content description to close review quality check-->
     <string name="browser_menu_review_quality_check_close" translatable="false">Close review quality check</string>
     <string name="review_quality_check" translatable="false">Review quality check</string>
+    <string name="review_quality_check_grade_a_b_description" translatable="false">Reliable reviews</string>
```

Comment:
Curious where these descriptors came from if you have a link. If they were to change, we would need separate vars for all five letters

Reply:
These are from the [component spec](https://www.figma.com/file/VE5xayz3yOSB5qOATUU9Im/Mobile-Shopping-Experience-MVP?type=design&node-id=2552-79863&mode=design&t=UoJFXrLsHa3hN4hh-0)